pkg/web_css/lib/src/_base.scss

```
@@ -46,12 +46,12 @@ body,
     font-weight: 400;
   }
 
-  h1 { font-size: 36px; }
-  h2 { font-size: 24px; }
-  h3 { font-size: 18px; }
-  h4 { font-size: 18px; }
-  h5 { font-size: 16px; }
-  h6 { font-size: 16px; }
+  h1 { font-size: 36px; } // note: github uses 2em, ~32px
+  h2 { font-size: 24px; } // note: github uses 1.5em, ~24px
+  h3 { font-size: 21px; } // note: github uses 1.25em, ~20px
+  h4 { font-size: 18px; } // note: github uses 1em, ~16px
+  h5 { font-size: 16px; } // note: github uses 0.875em, ~14px
+  h6 { font-size: 16px; } // note: github uses 0.85em, ~13.6px
```

Should we differentiate h6 now we are here? (Not sure we ever use h6 thought)
```suggestion
  h6 { font-size: 16px; } // note: github uses 0.85em, ~13.6px
```

---

The idea to have a headline text that is smaller than the paragraph text (even though it may be bolder) is a bit strange to me, but apparently others dig up its origins: https://stackoverflow.com/questions/55696808/why-do-h5-and-h6-have-smaller-font-sizes-than-p-in-most-user-agent-default
https://ux.stackexchange.com/questions/126039/is-it-a-problem-if-h4-h5-and-h6-are-smaller-than-regular-text

With the later, I think I'd keep the 16px, as our current `<p>` is 16px.